crates/kilt-dip-primitives/src/verifier/parachain/mod.rs

```diff
@@ -84,21 +93,31 @@ impl<
 		KiltWeb3Name,
 		KiltLinkableAccountId,
 		ConsumerBlockNumber,
-	> where
-	RelayBlockNumber: Default,
-	KiltDidKeyId: Default + Clone,
-	KiltAccountId: Clone,
-	KiltBlockNumber: Default + Clone,
-	KiltWeb3Name: Clone,
-	KiltLinkableAccountId: Clone,
-	ConsumerBlockNumber: Default,
-	Context: Clone,
+	>
 {
-	fn worst_case(context: Context) -> Self {
-		Self::V0(ParachainDipDidProof::worst_case(context))
+	fn from(
+		value: ParachainDipDidProof<
+			RelayBlockNumber,
+			KiltDidKeyId,
+			KiltAccountId,
+			KiltBlockNumber,
+			KiltWeb3Name,
+			KiltLinkableAccountId,
+			ConsumerBlockNumber,
+		>,
+	) -> Self {
+		Self::V0(value)
```

Comment:
If a new version is introduced, it will be because of a breaking change, so they will be different from each other. Otherwise if they cannot be distinguished, why would they be different in their versions?